style.css

```diff
@@ -0,0 +1,47 @@
+.square {
+  height: 150px;
+  width: 150px;
+  border: 1px solid #282c37;
+  margin: 5px;
+  text-align: center;
+  font-size: 120px;
+  font-family: 'Lato', sans-serif;
```

Comment:
If you specify the font-family on the `body`, it "trickles down" to the elements inside the body.